packages/aws-lambda-auto-wrap/esm/README.md

```diff
@@ -10,6 +10,15 @@ But AWS forgot to add support for layers. There are currently two known bugs:
 
 We are affected by the second bug. When AWS fixes the underlying problem in their AWS runtime, we are able to transform the new esm handler into a real ES module.
 
-For now: We ship the ES handler as commonjs module with the help of dynamic imports.
+For the time being, we are shipping the ES handler as a CommonJS module using dynamic imports as a workaround for Node.js 14 and 16, as AWS has yet to resolve the issue for these versions, which are no longer LTS.
+see the comment https://github.com/aws/aws-sdk-js-v3/issues/3230#issuecomment-1561973247
 
-In 4.x: We remove the aws-lambda-auto-wrap npm package and manually copy over the files in the publish layer script.
\ No newline at end of file
+The Node.js 18.x runtime introduces support for ES module resolution using NODE_PATH. For more information, refer to [Node.js 18.x runtime now available in AWS Lambda](https://aws.amazon.com/blogs/compute/node-js-18-x-runtime-now-available-in-aws-lambda/). This is the recommended workaround.
```

Comment:
I am not able to follow the connection between the [ES module resolution using NODE_PATH](https://aws.amazon.com/blogs/compute/node-js-18-x-runtime-now-available-in-aws-lambda/) and the Lambda layer  🤔 Could you please explain how that fixes the layer?

Refs:

> 2. You cannot define an ES module as handler, which is located in a layer. See AWS support case ID 11226031451.

> We are affected by the second bug. When AWS fixes the underlying problem in their AWS runtime, we are able to transform the new esm handler into a real ES module.

Did you test deploying our layer with the ESM handler with native ESM code? What was the result?
If yes and it was successful, what do we need to change?

Reply:
I haven't able to test this, will do it and update the status

Reply:
I'm getting an error like this with transftering the layer handler to pure ESM  "Error: Cannot find module 'instana-aws-lambda-auto-wrap-esm'\nRequire stack:\n- /var/runtime/index.mjs'

Reply:
Could you please share the steps you did?
If the esm handler would be a native handler, how would the handler look like and would we benefit from it?

Reply:
I will add details in the box note.

Reply:
Is this done? Could you send me the link in Slack? Thanks <3